ansible/roles/openshift_dedicated_admin/handlers/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+# handlers file for openshift_dedicated_admin
+handlers:
+- name: restart openshift-dedicated-role
+  service: 
+    name=openshift-dedicated-role 
+    state=restarted
```

Comment:
We prefer to use yaml syntax for everything possible. It makes the ansible error messages much better, and allows us to pass in complex arguments.

Consider changing this to:
```
handlers:
 - name: restart openshift-dedicated-role
   service: 
     name: openshift-dedicated-role 
     state: restarted
```